guides/release/getting-started/quick-start.md

```diff
@@ -83,6 +83,8 @@ Congratulations! You just created and booted your first Ember app.
 
 ## Write some HTML in a template
 
+<div data-template-tag="off" markdown="1">
```

Comment:
`markdown="1"` is required here to get `showdown` to continue parsing inside this tag. Without it you lose all support for formatting, etc.